code/modules/projectiles/projectile/homing_projectiles.dm

```diff
@@ -50,3 +50,43 @@
 		var/mob/living/carbon/human/H = target
 		var/obj/item/organ/external/E = pick(H.bodyparts)
 		E.add_embedded_object(T)
+
+/obj/item/projectile/homing/magic/homing_fireball
+	name = "greater bolt of fireball"
+	icon_state = "fireball"
+	damage = 20
+	damage_type = BRUTE
+	nodamage = FALSE
+
+	//explosion values
+	var/exp_devastate = 0
+	var/exp_heavy = 1
+	var/exp_light = 3
+	var/exp_flash = 4
+	var/exp_fire = 3
```

Comment:
not a fan of these, especially because they're all just labeled under exp

Reply:
fireball :)